### ISSUE TITLE
ver 2.0.0 : `undefined method `downcase'

### ISSUE THREAD

**Issue by kntmrkm** [fnando/browser#210]
Hi, 

I have below error with version 2.0.0.

`undefined method `downcase' for #<Hash:0x007ff2d3651850>`

```
    60: def downcased_ua
    61:   binding.pry
 => 62:   @downcased_ua ||= ua.downcase
    63: end

4.2.5.1@2.3.0 (#<Browser::Bot:0x007f9bc76e54a0>)> ua
=> {
    :ua => "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_11_3) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/48.0.2564.116 Safari/537.36"
}
```

Above code should be like `@downcased_ua ||= ua[:ua].downcase` ?

**Comment by fnando** [comment]
The constructor now has a different signature. It's `Browser.new(ua, options)`, where `ua` must be a string and `options` can have the `accept_language` header.

**Comment by kntmrkm** [comment]
OK, I found the problem.

See https://github.com/ankane/ahoy/issues/146.

Thanks.

**Comment by edslocomb** [comment]
I'm still getting this when running Rails tests, where apparently request.user_agent can be nil.

The line [here](https://github.com/fnando/browser/blob/master/lib/browser/middleware/context.rb#L11) needs to be `request.user_agent.to_s`

**Comment by fnando** [comment]
@edslocomb I commited a fix for this. I'll release this as soon as I also fix #212. In the meantime, use `gem "browser", github: "fnando/browser", branch: "master"`.
